project.clj

```diff
@@ -59,6 +59,7 @@
                  [puppetlabs/jruby-utils "0.1.0"]
                  [puppetlabs/trapperkeeper ~tk-version]
                  [puppetlabs/trapperkeeper-authorization "0.7.0"]
+                 [puppetlabs/trapperkeeper-scheduler "0.0.1"]
```

Comment:
I noticed this the other day as well.  I wouldn't sweat it too much ... if we want to do something about it, I would suggest just doing another release (probably 1.0 since we're using it in production now) of tk-scheduler - and hopefully get a tag out of it this time.